content/docs/setup/kubernetes/install/helm/index.md

```diff
@@ -193,19 +163,59 @@ to manage the lifecycle of Istio.
     58
     {{< /text >}}
 
-1. Install the `istio` chart:
+1. Install the `istio` chart. For a production environment use the recommended Helm values or customize as needed:
 
     {{< text bash >}}
-
     $ helm install istio.io/istio --name istio --namespace istio-system
+    {{< /text >}}
+
+    {{< tip >}}
+    To try the `bookinfo` demo, use Helm values customized for the `bookinfo` application:
+    {{< /tip >}}
 
+    {{< text bash >}}
+    $ helm install istio --name istio --values install/kubernetes/helm/istio/values-istio-demo.yaml --namespace istio-system
     {{< /text >}}
 
-1. Uninstall steps:
+## Uninstall
+
+1. If you installed using Option 1 with `helm template`, uninstall with this command:
 
     {{< text bash >}}
+    $ kubectl delete namespace istio-system
+    {{< /text >}}
 
+1. If you installed using Option 2 and `Tiller`, uninstall with these commands:
+
+    {{< text bash >}}
     $ helm delete --purge istio
     $ helm delete --purge istio-init
+    {{< /text >}}
+
+## Deleting CRDs and Istio Configuration
+
+{{< tip >}}
```

Comment:
Do not add conceptual information to the procedure. Please update the corresponding Concept page instead and link to it.